process/project-lifecycle-documents/SLSA_graduation_stage.md

```diff
@@ -0,0 +1,72 @@
+  * The SLSA specification is useful for both software producers and consumers: producers can follow SLSA’s guidelines to make their software supply chain more secure, and consumers can use SLSA to make decisions about whether to trust a software package.
+
+### Project adoption
+The project must be able to show adoption by multiple parties, which could be production deployments or substantial use by established open source communities, and demonstrate the value of that adoption to either the end users or the open source community.
+  * Since the release of SLSA 1.0 in April 2023, evidence of its adoption keeps growing. This includes support in build platform offerings from companies like [Google](https://cloud.google.com/build/docs/overview), [IBM](https://cloud.ibm.com/docs/devsecops?topic=devsecops-cd-devsecops-slsa), and [Red Hat](https://developers.redhat.com/products/trusted-software-supply-chain/overview), as well as support for [npm package provenance in GitHub](https://github.blog/security/supply-chain-security/introducing-npm-package-provenance/) and integration with Sigstore among others.
+
+### Release cadence
+The project must be able to show a consistent release cadence.
+  * https://github.com/slsa-framework/slsa/tags
+
+### Governance
+Projects must have documented project governance and be able to demonstrate that governance in action.
+  * https://github.com/slsa-framework/governance
+
+Have a defined and documented roadmap and annual goals for the project
+  * https://github.com/slsa-framework/slsa/projects?query=is%3Aopen
+
+Project has met at least 4 times over a period of at least 2 months since becoming incubating
+  * The SLSA project has basically been meeting every Monday since 2022! See https://slsa.dev/notes/community
+
+Implements, practices, and refines mature software development and release practices, such as adherence to semantic versioning, and having a declared policy for stable releases and backported fixes.
+  * The SLSA project follows the Community Specification Development Process and has a clearly documented policy about version management:
+    * https://github.com/slsa-framework/governance/blob/main/5._Governance.md
+    * https://github.com/slsa-framework/slsa/blob/main/CONTRIBUTING.md#slsa-versions-management
+
+Projects should harden their build systems in accordance with the SLSA Framework
+  * Not Applicable but we do use features such as branch protection, Dependabot, and Mend Renovate bot.
+
+### Security audit
+When applicable, projects must have completed a security audit through a third party and addressed audit findings and recommendations.
+  * Not Applicable.
+
+### Security Baseline
+
+The project meets all applicable Security Baseline requirements:
```

Comment:
Do these requirements need to be applied to our demos and reference implementations?

Reply:
This something that I think we should clarify. I think in SLSA's case it's the spec and the SLSA GitHub generator.

Reply:
I'd say that the SLSA Jenkins generator is another tool that needs to be included, only if for the relatively wide use of Jenkins.

Reply:
This application merely refers to the SLSA specification work, the rest is currently in a different project: [SLSA Tooling project](https://github.com/ossf/wg-supply-chain-integrity/blob/main/slsa-tooling.md). The separation dates from when the SLSA spec group was a SIG and it may actually make sense to regroup these into a single project moving forward but I'd rather not delay getting this application processed so let's keep them separated for now.